src/interfaces/operators/IInterestDistributor.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.5.0;
```

Comment:
Any specific reason why  pragma is defined like that ?

Reply:
Allows any integration to pull in these interfaces, using any Solidity version.